Emby.Server.Implementations/LiveTv/TunerHosts/BaseTunerHost.cs

```diff
@@ -83,7 +83,7 @@ public async Task<List<ChannelInfo>> GetChannels(bool enableCache, CancellationT
                 try
                 {
                     var channels = await GetChannels(host, enableCache, cancellationToken).ConfigureAwait(false);
-                    var newChannels = channels.Where(i => !list.Any(l => string.Equals(i.Id, l.Id, StringComparison.OrdinalIgnoreCase))).ToList();
+                    var newChannels = channels.Where(i => !list.Any(l => string.Equals(i.Id, l.Id, StringComparison.OrdinalIgnoreCase)));
```

Comment:
I don't know why I removed the ToList. Without it it will iterate over the combined list. I think we should just assume that GetChannels doesn't contain duplicates.
```suggestion
                    var newChannels = channels.Where(i => !list.Any(l => string.Equals(i.Id, l.Id, StringComparison.OrdinalIgnoreCase))).ToList();
```